src/tools/iron/client/application/tasks/list/iron_list_task.e

```diff
@@ -137,8 +137,8 @@ feature -- Execute
 									if a_iron.installation_api.is_package_installed (p.item) then
 										print (" [installed] ")
 									end
-									if l_has_conflict then
-										print (" [conflict!] ")
+									if l_has_conflict and then attached l_package_names.item (p.item.identifier) as l_earlier_package then
```

Comment:
I would do

if l_has_conflict then
    if attached l_package_names.item (p.item.identifier) as l_earlier_package then
        print (" [ignoring this package as it is also listed in the " + l_earlier_package.repository.location_string + " repository!] ")
    else
        print (" [Conflict!] ")
    end

otherwise, the conflict may not be reported... even if it is likely that l_earlier_package is set.